src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,578 @@
+      <arg id="1" name="ImageCodec" type="ImageCodecEnum" min="0x00" max="0x00"/>
+      <arg id="2" name="FrameRate" type="int16u"/>
+      <arg id="3" name="BitRate" type="int32u"/>
+      <arg id="4" name="MinResolution" type="VideoResolutionStruct"/>
+      <arg id="5" name="MaxResolution" type="VideoResolutionStruct"/>
+      <arg id="6" name="Quality" type="int8u" min="1" max="100"/>
+    </command>
+
+    <command code="0x08" source="server" name="SnapshotStreamAllocateResponse" optional="true" disableDefaultResponse="true">
+      <description>This command SHALL be sent by the device in response to the SnapshotStreamAllocate command, carrying the newly allocated snapshot stream identifier.</description>
+      <arg id="0" name="SnapshotStreamID" type="int16u"/>
+    </command>
+
+    <command code="0x09" source="client" name="SnapshotStreamDeallocate" optional="true">
+      <description>This command SHALL deallocate an snapshot stream on the camera, corresponding to the given snapshot stream identifier.</description>
+      <access op="invoke" privilege="administer"/>
+      <arg id="0" name="SnapshotStreamID" type="int16u"/>
+    </command>
+
+    <command code="0x0A" source="client" name="SetStreamPriority" optional="false">
+      <description/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command code="0x0B" source="client" name="CaptureSnapshot" optional="true">
+      <description>This command SHALL return a Snapshot from the camera.</description>
+      <access op="invoke" privilege="administer"/>
+      <arg id="0" name="SnapshotStreamID" type="int16u"/>
+      <arg id="1" name="RequestedResolution" type="VideoResolutionStruct"/>
+    </command>
+
+    <command code="0x0C" source="server" name="CaptureSnapshotResponse" optional="true" disableDefaultResponse="true">
+      <description>This command SHALL be sent by the device in response to the CaptureSnapshot command, carrying the requested snapshot.</description>
+      <arg id="0" name="Data" type="octet_string"/>
+      <arg id="1" name="ImageCodec" type="ImageCodecEnum" min="0x00" max="0x00"/>
+      <arg id="2" name="Resolution" type="VideoResolutionStruct"/>
+    </command>
+
+    <command code="0x0D" source="client" name="SetViewport" optional="true">
+      <description>This command SHALL set the viewport in all video streams.</description>
+      <arg id="0" name="Viewport" type="ViewportStruct"/>
+    </command>
+
+    <command code="0x0E" source="client" name="SetImageRotation" optional="true">
+      <description>The data fields for this command SHALL be as follows: </description>
+      <arg id="0" name="Angle" type="int16u" default="0" min="0" max="359"/>
+    </command>
+
+    <command code="0x0F" source="client" name="SetImageFlipHorizontal" optional="true">
+      <description>The data fields for this command SHALL be as follows: </description>
+      <arg id="0" name="OnOff" type="boolean"/>
```

Comment:
Again, "OnOff" is a bad name for a boolean.

And yes, the On/Off cluster has it.  We should not copy it.

Reply:
Changed to Enabled.